src/spacy/fortex/spacy/spacy_processors.py

```diff
@@ -575,23 +612,34 @@ def process_umls_entity_linking(linker, result, input_pack: DataPack):
 
     # get medical entity mentions and UMLS concepts
     for item in medical_entities:
-        entity = MedicalEntityMention(
-            input_pack, item.start_char, item.end_char
+        medical_entry_name = config.medical_onto_type
```

Comment:
looks like a bit overkill to pass in the whole config, how about just passing in the required values.